website/thaliawebsite/settings.py

```diff
@@ -944,3 +952,14 @@ def from_env(
         "auth",
     ],
 }
+
+MONEYBIRD_ADMINISTRATION_ID = 380475297324075003
```

Comment:
make this a os.environ.get
(also, what happens if we don't set it? Then `MONEYBIRD_ADMINISTRATION_ID` should be `None` and the whole sync should just be inactive if you ask me. Is this implemented? Or will it crash?)